modules/nw-http2-haproxy.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-operator.adoc
```

Comment:
While it's outside the scope, I've long wanted to see this assembly split up into user stories for each of these (HTTP/2, sharding, ect.). I haven't had time to delve deeper into it.

Reply:
Same! I had intended to do just that with some of the larger 4.5 additions, but I think we should do some networking docs bucket coordination. :) Something to look forward to...soon. :)